test/server.js

```diff
@@ -37,6 +37,11 @@ describe("Server", () => {
 	describe("WebSockets", () => {
 		let client;
 
+		before((done) => {
+			Helper.config.public = true;
```

Comment:
What is this for (I mean, apart from fixing broken tests)? And isn't that already set [here](https://github.com/thelounge/lounge/blob/e292ef2bed8017eb913e48c6a5548753d2f633a6/test/fixtures/.lounge/config.js#L5)?

Reply:
WebSocket tests do not intend to test for authentication, so they expect to operate on a public instance.

The problem is that I had to override the fixture settings for the LDAP tests, which are by definition incompatible with public instances. I don't know what's best, maybe it'd be better to have a different fixture for this? Any way, I saw that some other tests put their config assumptions in their `before()` hook, so I did the same. I think this is reasonable: as soon as a tests assumes some config lines, it specifies them.